packages/atomic/src/components/facets-v1/facet-value-checkbox/facet-value-checkbox.pcss

```diff
@@ -9,7 +9,7 @@
 }
 
 .value-checkbox + label {
-  @apply w-full flex pl-8 pr-2 py-2.5 text-on-background cursor-pointer truncate hover:bg-neutral-light group;
+  @apply w-full flex pl-8 pr-2 py-2.5 text-on-background cursor-pointer truncate hover:bg-neutral-light;
```

Comment:
JIT mode gave this error:
![image](https://user-images.githubusercontent.com/54454747/129105713-26b9209c-c229-49c3-aa8e-4eec0b3c9069.png)

Is the `group` class necessary here? @ThibodeauJF